client/my-sites/checkout/checkout-thank-you/redesign-v2/products/google-workspace-product.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import ThankYouProduct from 'calypso/components/thank-you-v2/product';
+import { getEmailManagementPath } from 'calypso/my-sites/email/paths';
+
+export type ThankYouGoogleWorkspaceProductProps = {
+	productFamily: string;
+	numberOfMailboxesPurchased?: number;
+	domainName: string;
+	siteSlug?: string;
+};
+
+export function ThankYouGoogleWorkspaceProduct( {
+	productFamily,
+	domainName,
+	numberOfMailboxesPurchased,
+	siteSlug,
+}: ThankYouGoogleWorkspaceProductProps ) {
+	const translate = useTranslate();
+	const emailManagementPath = getEmailManagementPath( siteSlug, domainName );
+
+	let details;
+	if ( numberOfMailboxesPurchased && numberOfMailboxesPurchased > 1 ) {
+		details = translate( '%(quantity)s mailboxes for %(domainName)s', {
+			args: { quantity: numberOfMailboxesPurchased, domainName },
+		} );
+	} else {
+		details = translate( 'For %(domainName)s', {
+			args: { domainName },
+		} );
```

Comment:
This will be problematic for localization. As we're not translating the full string `Google Workspace for {domainName}`, but only the second part of the string `for {domainName}`. 

We show the email address for TItan: https://github.com/Automattic/wp-calypso/pull/87098/files#diff-53cbc4b4fa018f1791d3890033deea890007647abb0ed461b7eec158f2a6378fR80-R82

Did we go with the `for {domainName}` approach because we don't know the mailbox name? If that's the case, I would suggest showing the quantity even for singular mailbox:
` mailbox for %(domainName)s`.

Reply:
>Did we go with the for {domainName} approach because we don't know the mailbox name?

Yep, for GW, we don't even store any mailbox name in path-query.

> I would suggest showing the quantity even for singular mailbox:

But, what if a user initially bought a few mailboxes, and we don't know about it. It would be incorrect to say "1 mailbox for", if we don't know the right number.